### PR TITLE
Updates to contributing section

### PR DESCRIPTION
**Pull request by malloryfreeberg**
## Done

- Added image describing dataset suitability
- Updated landing page for Contributing - Data
- Created 3 subpages under Contributing - Data
    - contributing-data-processing-results.md
    - contributing-data-suitability.md
    - contributing-expect-prepare.md

## To-do
- [ ] Content requires review from @gabsie and/or @adriennes 
- [x] New pages need to be added to siteMap.js file in the Data Portal Repository
- [ ] contributing-expect-prepare.md subpage needs more detail

### CODE REVIEW

File: content/contribute/data/contributing-data-suitability.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Currently, the HCA aims to define a baseline atlas of *healthy* cell types and states. The HCA recognizes that many groups work with samples from both healthy and diseased tissue. If consent allows, we are happy to accept a dataset that contains data from both healthy and diseased tissue, which may provide value to the Atlas.
+
+### Organism
+
+The HCA intends to collect, process, and share data that comprehensively represents all *human* tissues. However, mouse models can provide access to some sample types and experimental strategies which are currently not possible in human. The current HCA data policy allows both human and mouse data to be submitted, but at this time, we are not accepting data from other species.
+
+### Data Processing Pipeline Support
+
+The HCA currently *supports* processing of human 10x 3' v2 and paired-end Smart-seq2 scRNA-seq data. We are currently working to establish processing pipelines for single nuclei RNA sequencing and image-based transcriptomics data as well as for mouse data. We are happy to accept datasets and subsets of datasets that we cannot currently analyze to help us build and prioritize future pipelines.
```
Comment (malloryfreeberg) [inline content/contribute/data/contributing-data-suitability.md]: @kbergin @TimothyTickle @jkaneria can you whether this text is correct regarding what pipelines are currently being worked on? Or let me know what should be changed? Thanks!

File: content/contribute/data/contributing-expect-prepare.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+path: "/contribute/data/contributing-expect-prepare"
+date: "2019-04-29"
+title: "Data Submission Process"
+---
+
+# Data Submission Process
```
Comment (malloryfreeberg) [inline content/contribute/data/contributing-expect-prepare.md]: @gabsie this page needs the most work in terms of adding content.

File: siteMap/siteMap.yaml [hunk 3]
```diff
@@ -177,6 +177,16 @@
         - name: "Contributing Data"
           key: "/contribute/data/contributing-to-hca"
           path: "/contribute"
+          secondaryLinks:
```
Comment (malloryfreeberg) [inline siteMap/siteMap.yaml]: @NoopDog I updated the siteMap.yaml file (which I think I'm supposed to do) to what I think it should be, but I'm not confident it's right!

**Comment by NoopDog** [comment]
will take a look at the sitemap.  Thanks Looks like this PR has conflicts tho.

**Comment by malloryfreeberg** [comment]
> Looks like this PR has conflicts tho.

Where do you see conflicts? The review looks green to me:

Screen Shot 2019-04-30 at 16 32 07

**Comment by NoopDog** [comment]
https://github.com/HumanCellAtlas/data-portal-content/pull/278
hmmm

Screen Shot 2019-04-30 at 8 33 51 AM

**Comment by NoopDog** [comment]
OK @malloryfreeberg I see how I can merge this with out conflicts.

@frano-m can you take a look at the site map and adjust the paths to the new contribute docs so that the left nav stays open when any of the new docs are selected. See how this does not open up the left nav:

http://localhost:8000/contribute/contributing-data-suitability.md

Thanks

**Comment by NoopDog** [comment]
Hi @malloryfreeberg the sitemap has been updated. Sorry about the force push, but we had merge conflicts and needed to rebase this on master before continuing. Next time we will push to a new branch instead of force pushing yours. There were a few updates made to internal links, image urls etc. to get  things working. Its looking good now locally. Is there further review to do here or are we ready to merge? Cheers, D

**Comment by malloryfreeberg** [comment]
@NoopDog no worries! Thanks for fixing the links.

I would like to get a review from @kbergin @TimothyTickle @jkaneria and @gabsie @adriennes about the content. Can we please wait to merge until after they have reviewed?

**Comment by NoopDog** [comment]
No wakas. Cheers, D

File: content/contribute/data/contributing-data-suitability.md (hunk 1 above)
Comment (kbergin) [reply content/contribute/data/contributing-data-suitability.md]: > We are currently working to establish processing pipelines for droplet based single nuclei RNA-seq, single end Smart-seq2 scRNA-seq, 10x 3' v3 scRNA-seq, support for mouse data for 10x 3' and Smart-seq2 scRNA-seq, and image-based transcriptomics data.

File: content/contribute/data/contributing-data-processing-results.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+# Data Processing and Results
+
+Once submitted to the Human Cell Atlas, your data will be processed and made available in multiple ways. 
+ 
+## Searchable in the HCA Data Browser
+
+Your raw data and metadata will be searchable in the [HCA Data Browser](/explore/projects) immediately after submission.
+
+## Processed by the data processing pipelines
```
[the remark this answers is not in the document]
Comment (malloryfreeberg) [reply content/contribute/data/contributing-data-processing-results.md]: Good point. Done!

File: content/contribute/data/contributing-data-suitability.md (hunk 1 above)
Comment (malloryfreeberg) [reply content/contribute/data/contributing-data-suitability.md]: Updated with Kylee's suggestion!

@ambrosejcarr do you want to add any more detail here about image-based assays? Here is the new text:

> The HCA currently *supports* processing of human 10x 3' v2 and paired-end Smart-seq2 scRNA-seq data. We are currently working to establish processing pipelines for droplet-based single nuclei RNA-seq, single end Smart-seq2 scRNA-seq, 10x 3' v3 scRNA-seq, support for mouse data for 10x 3' and Smart-seq2 scRNA-seq, and image-based transcriptomics data. We are happy to accept datasets and subsets of datasets that we cannot currently analyze to help us build and prioritize future pipelines.

**Comment by malloryfreeberg** [comment]
I have addressed Tim and Kylee's comments, but I haven't heard any feedback on these changes from @gabsie or @adriennes .  @NoopDog  can you please merge this PR when you have time? I would do it myself, but my "Merge pull request" button is green and the last time we discussed merging, it appeared to have merge conflicts on your end. 

We can incorporate additional feedback in future rounds of content review. Thanks!

**Pull request merged**